### ISSUE TITLE
[4.4.0-beta.3]: Source path component display glitch

### ISSUE THREAD

**Issue by thupsi** [craftcms/cms#12757]
### Description
The new "source path" component used in assets indexes does not display correctly when viewed on screens with scaling set to other than 100% or multiples of 100%.

Specifically there seem to be some slight "rounding" errors when calculating CSS sizes, resulting in a "broken" appearance, like this:

![image](https://user-images.githubusercontent.com/19857599/221805891-d54bdb53-ba7f-496a-b245-03ab8801a16f.png)

I have this issue on Windows 11 in all major browsers (Chrome, Edge, Firefox).

### Steps to reproduce

1. Be on windows 🪟
2. Have a 4K display that is not large enought to be used with no scaling 👓
3. Go to display settings and set the scaling to something like 150% or 125%.
4. Go to the asset index in the CP

### Expected behavior
The new source path component displays in all its glory.

### Actual behavior
The server crashes. Just kidding. The source path component is rendered incorrectly.

### Craft CMS version

4.4.0-beta.3

### Installed plugins and versions

-

**Comment by thupsi** [comment]
I just did some testing and found that this can be fixed by changing

```
.element-index .source-path .chevron-btns {
    --ui-control-height: 1.875rem;
    background: #fff;
    display: -webkit-inline-flex;
    display: inline-flex;
    position: relative;
}
```
to

```
.element-index .source-path .chevron-btns {
    --ui-control-height: 1.9rem;
    background: #fff;
    display: -webkit-inline-flex;
    display: inline-flex;
    position: relative;
}
```

**Comment by thupsi** [comment]
So, I took a look at the relevant commits and found that this stems from switching from `px` to `rem` for `--ui-control-height`.

While this is a good idea, it can be problematic when the final/calculated pixel value is not a whole number, as demonstrated in the above issue. Not sure of the recommended approach here, other than fixing bugs as/when they are discovered.

**Comment by gcamacho079** [comment]
Thank you for reporting this issue, @thupsi. I’ve created a PR to resolve this. 👍🏼

**Comment by brandonkelly** [comment]
3.8.0-beta.3 and 4.4.0-beta.4 are out with the fix for this, via #12766.
